_posts/tr/pages/2016-01-01-about.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Hakkında
+name: about
+permalink: /tr/about/
+type: pages
+layout: page
+lang: tr
+version: 1
```

Comment:
I think the version change you made in https://github.com/bitcoin-core/bitcoincore.org/commit/131d1219447574e3d7acee5c8233d75dfc69fe19#r64151455
applies here and maybe to other documents.

Reply:
As I said, I translated the original version 5 from English.

Reply:
5 does not exist for this document, last is version 3, see:

https://github.com/bitcoin-core/bitcoincore.org/blob/fabd138905877a23d582444e0487b187b2cb2f07/_posts/en/pages/2016-01-01-about.md#L8

Reply:
> 5 does not exist for this document, last is version 3, see:
> 
> https://github.com/bitcoin-core/bitcoincore.org/blob/fabd138905877a23d582444e0487b187b2cb2f07/_posts/en/pages/2016-01-01-about.md#L8

It was replaced with the version number of the English language version that I translated. (118a8de)